examples/server/tests/features/results.feature

```diff
@@ -13,6 +13,7 @@ Feature: Results
 
   Scenario Outline: consistent results with same seed
     Given <n_slots> slots
+    And   0.0 temperature
```

Comment:
This should be set to temperature 1, not temperature 0. With temperature 0 you get greedy decoding in which case the seed is irrelevant. So this test can no longer detect issues with e.g. the RNG state being shared between slots.

Reply:
Ah yes, will push a fix now